tests/x11regressions/gnomecase/gnome_default_applications.pm

```diff
@@ -61,7 +61,7 @@ sub run() {
     send_key "ctrl-w";                                     #close fileroller
     wait_still_screen;
     assert_and_dclick "gnomecase-defaultapps-htmlfile";    #open html
-    assert_screen 'gnomecase-defaultapps-firefoxopen';
+    assert_screen 'gnomecase-defaultapps-firefoxopen', 60;
```

Comment:
On Monday 30 May 2016 23:22:17 chuchingkai wrote:
> > -    assert_screen 'gnomecase-defaultapps-firefoxopen';
> > +    assert_screen 'gnomecase-defaultapps-firefoxopen', 60;
> 
> @okurz My local server has a low performance, and the default timeout (30 s)
> is not enough sometimes.

try setting setting the TIMEOUT_SCALE variable for your worker, e.g. 
TIMEOUT_SCALE=2
This variable is used for slow workers.